web/src/features/prompts/server/routers/promptRouter.ts

```diff
@@ -635,6 +635,7 @@ export const promptRouter = createTRPCRouter({
           WHERE
               o.type = 'GENERATION'
               AND s.data_type != 'CATEGORICAL'
+              AND s.value IS NOT NULL
```

Comment:
do you have an issue with quick follow ons regarding this (need for displaying categorical scores (value counts) on tables

Reply:
Yes: https://linear.app/langfuse/issue/LFE-1270/add-categorical-score-counts-averages-in-charts-prompt-metrics